src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -98,24 +103,51 @@ class CodeLongevity(private val localRepo: LocalRepo,
         // TODO(anatoly): Add emails from server or hashAll.
         val emails = hashSetOf(localRepo.author.email)
 
-        val sum: MutableMap<String, Long> = emails.associate { Pair(it, 0L) }
-                                                  .toMutableMap()
-        val total: Int = codeLines.size
+        val sums: MutableMap<String, Long> = emails.associate { Pair(it, 0L) }
+                                                   .toMutableMap()
+        val totals: MutableMap<String, Int> = emails.associate { Pair(it, 0) }
+                                                    .toMutableMap()
+
+        val repoTotal: Int = codeLines.size
+        var repoSum: Long = 0
         for (line in codeLines) {
+            repoSum += line.age
+
             val email = line.from.commit.authorIdent.emailAddress
             if (!emails.contains(email)) {
                 continue
             }
             Logger.debug(line.toString())
             Logger.debug("Age: ${line.age} secs")
 
-            sum[email] = sum[email]!! + line.age
+            sums[email] = sums[email]!! + line.age
+            totals[email] = totals[email]!! + 1
         }
 
+        val repoAvg = if (repoTotal > 0) { repoSum / repoTotal } else 0
+        val stats = mutableListOf<Fact>()
+        stats.add(Fact(repo = serverRepo,
+                       key = CODE_LONGEVITY_REPO_KEY,
+                       value = repoAvg.toDouble()))
+        println("Repo average code line age is $repoAvg seconds, "
```

Comment:
Shouldn't this be log? And why seconds? It should be days at least :)

Reply:
Yep, you're right.